commands/cfe.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const Validation = require('../utils/validator');
+const myValidator = new Validation();
+const sendMessage = require('../utils/sendTableMessage');
+const Scrapper = require('../webScraping/cfeScrapper');
+
+/**
+ * Main function of the command
+ * @param { Message } message Channel to send the data
+ * @arg { searchItems } args Searching parameters
+ * @return { void }
+ */
+async function execute(message, args) {
+  if (!args || args.length == 0) {
+    return message.channel.send('The command needs a searching parameter.');
+  }
+  const route = `./assets/cfe_${args.join('').toLowerCase()}.json`;
+  args = args.join(' ');
+  const executeScrapper = myValidator.isFileLastUpdateIn(route);
+  if (!executeScrapper) {
+    message.reply('Is needed execute the scrapper, executing...');
```

Comment:
Nit: I think the copy here is not correct. What is needed? Need to specify what are you referring too